### PR TITLE
Fix mandelbug in Test Locked Cover

### PR DESCRIPTION
**Pull request by idgserpro**
Wait for the page load. Avoid:

https://travis-ci.org/collective/collective.cover/jobs/416408698#L1766

### CODE REVIEW

File: src/collective/cover/tests/test_locked_cover.robot [hunk 1]
```diff
@@ -43,6 +43,7 @@ Test Locked Cover
     Enable Autologin as  Site Administrator
     Goto Homepage
     Click Link  link=My Cover
+    Wait Until Page Contains  Locked
```
Comment (hvelarde) [inline src/collective/cover/tests/test_locked_cover.robot]: why note replace the whole thing?

```diff
-Page Should Contain  Locked  ${LOCKED_MESSAGE}
+Wait Until Page Contains  Locked  ${LOCKED_MESSAGE}
```

File: src/collective/cover/tests/test_locked_cover.robot [hunk 2]
```diff
@@ -56,6 +57,7 @@ Test Locked Cover
 
     Switch Browser  1
     Click Link  link=My Cover
+    Wait Until Page Contains  Locked
```
Comment (hvelarde) [inline src/collective/cover/tests/test_locked_cover.robot]: why note replace the whole thing?

```diff
-Page Should Contain  Locked  ${LOCKED_MESSAGE}
+Wait Until Page Contains  Locked  ${LOCKED_MESSAGE}
```

File: src/collective/cover/tests/test_locked_cover.robot [hunk 3]
```diff
@@ -69,6 +71,7 @@ Test Locked Cover
 
     Switch Browser  2
     Click Link  link=My Cover
+    Wait Until Page Contains  Locked
```
Comment (hvelarde) [inline src/collective/cover/tests/test_locked_cover.robot]: why note replace the whole thing?

```diff
-Page Should Contain  Locked  ${LOCKED_MESSAGE}
+Wait Until Page Contains  Locked  ${LOCKED_MESSAGE}
```

File: src/collective/cover/tests/test_locked_cover.robot [hunk 4]
```diff
@@ -82,4 +85,5 @@ Test Locked Cover
 
     Switch Browser  1
     Click Link  link=My Cover
+    Wait Until Page Contains  Locked
```
Comment (hvelarde) [inline src/collective/cover/tests/test_locked_cover.robot]: why note replace the whole thing?

```diff
-Page Should Contain  Locked  ${LOCKED_MESSAGE}
+Wait Until Page Contains  Locked  ${LOCKED_MESSAGE}
```

File: src/collective/cover/tests/test_locked_cover.robot (hunk 2 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: ```Wait Until Page Contains``` doen't support a second term for comparison.

I'll only use ```$ {LOCKED_MESSAGE}```

File: src/collective/cover/tests/test_locked_cover.robot (hunk 3 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: ```Wait Until Page Contains``` doen't support a second term for comparison.

I'll only use ```$ {LOCKED_MESSAGE}```

File: src/collective/cover/tests/test_locked_cover.robot (hunk 4 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: ```Wait Until Page Contains``` doen't support a second term for comparison.

I'll only use ```$ {LOCKED_MESSAGE}```

File: src/collective/cover/tests/test_locked_cover.robot (hunk 1 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: ```Wait Until Page Contains``` doen't support a second term for comparison.

I'll only use ${LOCKED_MESSAGE}

Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: Done.

File: src/collective/cover/tests/test_locked_cover.robot (hunk 2 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: Done.

File: src/collective/cover/tests/test_locked_cover.robot (hunk 3 above)
Comment (idgserpro) [reply src/collective/cover/tests/test_locked_cover.robot]: Done.

**Comment by idgserpro** [comment]
In fact, ```Page Should Contain``` and ```Page Should Not Contain``` also don't support a second parameter. They were simply ignored.

I had to change ```${LOCKED_MESSAGE}``` because ```Wait Until Page Contains``` has problems with "formatted strings".

In ```This item was locked by admin 1 minute ago.```, ```admin``` is a link. ```Wait Until Page Contains``` cann't find this link text.

**Pull request merged**